conformance_tests/sysman/test_sysman_power/src/test_sysman_power.cpp

```diff
@@ -979,7 +1169,7 @@ TEST_F(
                 zesPowerGetProperties(p_power_handle, &pProperties));
 
       EXPECT_GT(pExtProperties.domain, ZES_POWER_DOMAIN_UNKNOWN);
-      EXPECT_LE(pExtProperties.domain, ZES_POWER_DOMAIN_STACK);
+      EXPECT_LE(pExtProperties.domain, ZES_POWER_DOMAIN_GPU);
```

Comment:
Why this change ?

Reply:
Its the order in which the enum is defined. GPU can get enumerated now, like in BMG so we should check the value to be lesser than GPU

```
typedef enum _zes_power_domain_t
{
    ZES_POWER_DOMAIN_UNKNOWN = 0,                                           ///< The PUnit power domain level cannot be determined.
    ZES_POWER_DOMAIN_CARD = 1,                                              ///< The PUnit power domain is a card-level power domain.
    ZES_POWER_DOMAIN_PACKAGE = 2,                                           ///< The PUnit power domain is a package-level power domain.
    ZES_POWER_DOMAIN_STACK = 3,                                             ///< The PUnit power domain is a stack-level power domain.
    ZES_POWER_DOMAIN_MEMORY = 4,                                            ///< The PUnit power domain is a memory-level power domain.
    ZES_POWER_DOMAIN_GPU = 5,                                               ///< The PUnit power domain is a GPU-level power domain.
    ZES_POWER_DOMAIN_FORCE_UINT32 = 0x7fffffff

} zes_power_domain_t;
```

Reply:
I am not sure why only till STACK was used previously, if the spec had references till GPU

We should be using EXPECT_LT (ZES_POWER_DOMAIN_FORCE_UINT32)

Reply:
updated this